includes/Checker/AJAX_Runner.php

```diff
@@ -45,27 +44,22 @@ public function is_plugin_check() {
 	}
 
 	/**
-	 * Creates and returns an instance of the Checks class based on the request.
+	 * Returns the plugin slug based on the request.
 	 *
 	 * @since n.e.x.t
 	 *
-	 * @return Checks An instance of the Checks class.
+	 * @return string The plugin slug.
 	 *
-	 * @throws Exception Thrown if the plugin main file cannot be found based on the AJAX input.
+	 * @throws Exception Thrown if the plugin slug is invalid.
 	 */
-	protected function get_checks_instance() {
-		if ( ! isset( $this->checks ) ) {
-			if ( ! isset( $_REQUEST['plugin'] ) ) {
-				throw new Exception( 'Invalid plugin slug: Plugin slug must not be empty.' );
-			}
-
-			// Get the plugin name from the AJAX request.
-			$plugin_file = Plugin_Request_Utility::get_plugin_basename_from_input( $_REQUEST['plugin'] );
-
-			$this->checks = new Checks( WP_PLUGIN_DIR . '/' . $plugin_file );
+	protected function get_plugin_param() {
+		if ( ! isset( $_REQUEST['plugin'] ) ) {
+			throw new Exception(
+				__( 'Invalid plugin slug: Plugin slug must not be empty.', 'plugin-check' )
```

Comment:
See other similar comment, can you adjust the exception message to refer to "plugin parameter" rather than "plugin slug"?